src/Stomp/Client.php

```diff
@@ -154,6 +160,28 @@ public function setVhostname($host = null)
         $this->host = $host;
     }
 
+  /**
+   * Set the desired heartbeat for the connection.
+   *
+   * A heartbeat is a specific message that will be send / received when no other data is send / received
+   * within an interval - to indicate that the connection is still stable. If client and server agree on a beat and
+   * the interval passes without any data activity / beats the connection will be considered as broken and closed.
+   *
+   * If you define a heartbeat, you must assure that your application will send data within the interval.
```

Comment:
hmm isn't it exactly the opposite? in case you use heartbeats you dont need to send data in certain intervalls as the client will automatically sent a hearbeat to keep the connection alive..?